aocs/parser.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
```

Comment:
i will merge this parser but be careful for the next time, you add on your commit this old aocs parser. i will delete the old change in master branch